Configuration/Services.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+return static function (ContainerConfigurator $containerConfigurator) {
+    $services = $containerConfigurator->services()
+        ->defaults()
+        ->autowire()
+        ->autoconfigure()
+    ;
+
+    $services->load('TTN\\Tea\\', '../Classes/*');
```

Comment:
The exclude is missing from Services.yaml

Reply:
Thanks, good catch. I've added the exclude.